tests/unit/checks/test_ibm_mq_queues.py

```diff
@@ -519,3 +525,57 @@ def assert_procs(opprocs, params, expected):
     }
     actual = list(check.run_check('QM1:MY.QUEUE', params, parsed))
     assert expected == actual[1]
+
+
+#
+# QTIME
+#
+
+
+@pytest.mark.usefixtures("config_load_all_checks")
```

Comment:
This line needs to be dropped.
There used to be nothing wrong with it, but I removed the fixture yesterday. All plugin loading fixtures have been refactored, and since virtually every test under tests/unit/checks needs them, it's an autouse fixture now.
See git log of tests/unit/conftest.py and tests/unit/checks/conftest.py for details.